coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE liquidity_requests (
+    id SERIAL PRIMARY KEY,
+    trader_pk TEXT NOT NULL,
+    timestamp TIMESTAMP WITH TIME ZONE NOT NULL NOT NULL,
+    requested_amount_sats BIGINT NOT NULL,
+    liquidity_option TEXT NOT NULL,
```

Comment:
Once #1384 is in this should be a reference to the `id` of said liquidity option.